scaaml/capture/scope/ps6424e.py

```diff
@@ -310,6 +351,15 @@ def _dict_repr(self):
         ret["trigger_direction"] = self.trigger_direction
         return ret
 
+    @property
+    def is_digital(self) -> bool:
+        """Return True iff this channel is a digital channel (PORT0 or PORT1).
```

Comment:
typo or is the double "f" voluntary?

Reply:
iff == if and only if (sorry my math talk :))